stories/s1grdiw.eocloud.stories.js

```diff
@@ -146,19 +140,16 @@ export const getMapWMSEvalscript = () => {
   wrapperEl.insertAdjacentElement('beforeend', img);
 
   const perform = async () => {
-    const layer = new S1GRDEOCloudLayer(
+    const layer = new S1GRDEOCloudLayer({
       instanceId,
       layerId,
-      `
+      evalscript: `
         return [2.5 * VV, 1.5 * VV, 0.5 * VV];
       `,
-      null,
-      null,
-      null,
-      AcquisitionMode.IW,
-      Polarization.DV,
-      Resolution.HIGH,
-    );
+      acquisitionMode: AcquisitionMode.IW,
+      polarization: Polarization.DV,
+      resolution: Resolution.HIGH,
```

Comment:
repeated just for easy find: resolution isn't a parameter for S1GRDEOCloudLayer anymore